lib/test/test_help_header.rb

```diff
@@ -7,25 +7,31 @@ def assert_includes_id(items, id)
     assert items.find {|e| e[:id] == id}
   end
 
-  def test_hamburger_content_script_level
+  def test_help_header_content_script_level
```

Comment:
Did some test renaming here, I think `help_header` is more appropriate name?

Reply:
Oh yes -- thank you!